Fixes the build with CMake 3.4.0.

------------------------------------------------------------------------
r191042 | commit-queue@webkit.org | 2015-10-14 12:57:49 +0200 (Wed, 14 Oct 2015) | 12 lines

[GTK][EFL] Fix build with cmake 3.4
https://bugs.webkit.org/show_bug.cgi?id=150117

Explicitely include the CheckIncludeFiles module before using
the CHECK_INCLUDE_FILES command.

Patch by Tomas Popela <tpopela@redhat.com> on 2015-10-14
Reviewed by Žan Doberšek.

* Source/cmake/FindOpenGL.cmake:
* Source/cmake/FindWebP.cmake:
* Source/cmake/OptionsEfl.cmake:
------------------------------------------------------------------------

Index: /trunk/Source/cmake/FindOpenGL.cmake
===================================================================
--- Source/cmake/FindOpenGL.cmake	(revision 191041)
+++ Source/cmake/FindOpenGL.cmake	(revision 191042)
@@ -60,4 +60,5 @@
     CMAKE_PUSH_CHECK_STATE()
     set(CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES} ${OPENGL_INCLUDE_DIRS})
+    include(CheckIncludeFiles)
     check_include_files("GL/glx.h" GLX_FOUND)
     CMAKE_POP_CHECK_STATE()
Index: /trunk/Source/cmake/FindWebP.cmake
===================================================================
--- Source/cmake/FindWebP.cmake	(revision 191041)
+++ Source/cmake/FindWebP.cmake	(revision 191042)
@@ -36,4 +36,5 @@
     # Older installations of libwebp do not install a pkgconfig file, so we fall
     # back to a manual search for a libwebp header file.
+    include(CheckIncludeFiles)
     check_include_files("webp/decode.h" WEBP_FOUND_HEADER)
     if (WEBP_FOUND_HEADER)
